src/types.ts

```diff
@@ -80,6 +82,13 @@ export interface ActorTool extends ToolBase {
 export type InternalToolArgs = {
     /** Arguments passed to the tool */
     args: Record<string, unknown>;
+    /** Extra data given to request handlers.
+     *
+     * Can be used to send notifications from the server to the client.
+     *
+     * For more details see: https://github.com/modelcontextprotocol/typescript-sdk/blob/f822c1255edcf98c4e73b9bf17a9dd1b03f86716/src/shared/protocol.ts#L102
+     */
```

Comment:
```suggestion
     * For more details see: https://github.com/modelcontextprotocol/typescript-sdk/blob/f822c1255edcf98c4e73b9bf17a9dd1b03f86716/src/shared/protocol.ts
     */
```
I would remove the link to a specific line of code. It'll be obsolete. Sooner or later, someone will update this file, and it will point to the wrong line. So it is better to provide less specific reference than a wrong one.

Reply:
This is why the link is blob, it should thus point to the specific, pinned, version of file